src/context/WishlistContext.jsx

```diff
@@ -0,0 +1,20 @@
+import {
+  createContext,
+  useContext,
+  useState,
+} from "react/cjs/react.development";
+
+const WishlistContext = createContext();
+
+export function WishlistProvider({ children }) {
+  const [wishList, setWishList] = useState([]);
+  return (
+    <WishlistContext.Provider value={{ wishList, setWishList }}>
+      {children}
+    </WishlistContext.Provider>
+  );
+}
+
```

Comment:
Suggestion : Do Add API Calls in the context , then call those functions wherever required!!